src/components/ContentWidgets/Separator/SeparatorEditor.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { Separator } from "./Separator";
+
+/**
+ * SeparatorEditor edits the width and color of a separator widget
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 10, 2019
+ */
+export const SeparatorEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  const defaultColor = { r: 66, g: 13, b: 171 };
+  const width = editedContent.separator_width || 33;
```

Comment:
Set all these values, `33`, `"red"`, `"green"`, `"blue"` out into constants defined either above outside of this function or in some sort of constants file. That way we aren't reliant on just strings.

Actually, for color I would do like an enum or something instead.

```tsx
enum Color {
  RED = "RED", 
  GREEN = "GREEN", 
  BLUE = "BLUE"
}